packages/react/src/components/Toggle/Toggle-story.js

```diff
@@ -6,28 +6,9 @@
  */
 
 import React from 'react';
-import { action } from '@storybook/addon-actions';
-import { withKnobs, text, boolean, select } from '@storybook/addon-knobs';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
 import Toggle from '../Toggle';
-
```

Comment:
Can we add a Playground story so users can play around with the knobs? Would that be enough to handle the `small` variant, or do you think we should break that into a separate story?

Reply:
Good question! We can. I just added separate stories since I know we're trying to make things copy/paste-able off the bat. I thought it would help with that. But can def add a playground story. I don't have a strong preference either way.

Reply:
Yeah definitely! Then let's keep the current stories so that they are copy/paste-able and just add in one with all the works 👍🏻